lib/cn/cn.c

```diff
@@ -1522,9 +1522,6 @@ cn_make(struct mpool *ds, const struct kvs_cparams *cp, struct kvdb_health *heal
     if (cp->fanout < CN_FANOUT_MIN || cp->fanout > CN_FANOUT_MAX)
         return merr(EINVAL);
 
-    if (cp->fanout != roundup_pow_of_two(cp->fanout))
-        return merr(EINVAL);
-
```

Comment:
Rename this PR to reflect this change?

Reply:
OOPS. didn't mean to include that. Not ready yet since the jury is still out. Good catch

Reply:
I'll get to it soon!